yesod-test/ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # ChangeLog for yesod-test
 
+## TODO
```

Comment:
Not sure what version stuff we're following. Adding a module is usually a minor bump, but this is an internal module, so patch may be appropriate.

Reply:
I'd still like to use minor so that version bounds and CPP can easily depend on it.